src/main/java/core/basesyntax/models/RightTriangle.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.models;
+
+import core.basesyntax.Figure;
+
+public class RightTriangle implements Figure {
+    private final String color;
+    private final double firstLeg;
+    private final double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        this.color = color;
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Color: " + color

```

Comment:
There is a missing comma in the `println` statement, which can lead to concatenated and unclear output. Consider adding a comma after the color value to separate the attributes for better readability.